### DISCUSSION TITLE
WordPress Theme Development

### DISCUSSION THREAD

**Discussion by black-dots** [WordPress/developer-blog-content#59] [Q&A]
I am new here. I want to be a WordPress theme developer. Please give me a proper guideline to be a good WordPress Theme Developer?

**Comment by bph** [comment]
Sorry for the delayed answer, @black-dots 

A good source to start would be the published [Block theme resources roundup](https://developer.wordpress.org/news/2023/02/block-theme-resources-roundup/) with plenty of resources for Block Theme development, especially the links to the [Learn.WordPress](https://learn.wordpress.org) site
